src/core/issues-notes.js

```diff
@@ -141,31 +141,35 @@ export function run(conf, doc, cb) {
     issueBase = conf.issueBase;
   if ($ins.length) {
     if (conf.githubAPI) {
-      ghFetch(conf.githubAPI)
-        .then(function(json) {
-          issueBase = issueBase || json.html_url + "/issues/";
-          return fetchIndex(json.issues_url, {
-            // Get back HTML content instead of markdown
-            // See: https://developer.github.com/v3/media/
-            Accept: "application/vnd.github.v3.html+json",    
-          });
-        })
-        .then(function(issues) {
-          issues.forEach(function(issue) {
-            ghIssues[issue.number] = issue;
-          });
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        }).catch(err => {
-          pub("error", err.message);
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        });
+        fetchFunction(conf.githubAPI);
```

Comment:
See comments I left in https://github.com/w3c/respec/pull/1534#discussion_r172388979 about try/catch/finally... would probably apply here too.

Reply:
Makes sense! I will update that and push again tomorrow, since I'm in a hurry right now. Thanks for the hint